ios/KaMPKitiOS/AppDelegate.swift

```diff
@@ -13,10 +13,23 @@ import shared
 class AppDelegate: UIResponder, UIApplicationDelegate {
     
     var window: UIWindow?
-    let log = KoinIOS().get(objCClass: Kermit.self, parameter: "AppDelegate") as! Kermit
-    
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        // Override point for customization after application launch.
+
+    // Lazy so it doesn't try to initialize before startKoin() is called
+    lazy var log = koin.get(objCClass: Kermit.self, parameter: "AppDelegate") as! Kermit
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions
+        launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+
+        startKoin()
+              
+        // Manually launch storyboard so that ViewController doesn't initialize before Koin
+        let storyboard = UIStoryboard.init(name: "Main", bundle: nil)
```

Comment:
For increased Swiftiness, you should always be able to get rid of 'init' from initializers: `UIStoryboard(name: "Main", bundle: nil)`is usually seen as more elegant and readable, even if it's technically less verbose.

Reply:
Yeah that's a bad habit of mine I should break